src/patternfly/components/Form/form.scss

```diff
@@ -106,6 +110,15 @@
   }
 }
 
+.pf-c-form__section {
+  display: grid;
+  gap: var(--pf-c-form__section--Gap);
+
+  &:not(:last-child) {
```

Comment:
It seems valid that all form control blocks could be sections, meaning you could end up with a block that looks like this, and you have too much space between the last section and the action group.

<img width="1435" alt="Screen Shot 2020-09-16 at 11 10 11 AM" src="https://user-images.githubusercontent.com/35148959/93363801-5d315980-f80d-11ea-9407-a8ab0a2d07fd.png">

Maybe this should be `&:not(:first-child) { margin-top... }`?

Reply:
Updated